### ISSUE TITLE
Help Needed

### ISSUE THREAD

**Issue by vikramvi** [kaygee/timings-client#1]
@kaygee Marcel from Godaddy has informed that you're interacting with him about timings api and asked queries.

It'll be really helpful if you can put your queries and answers under https://github.com/Verkurkie/timings-docker/issues/13.

Even I'm using timings project in my company. Thanks in advance.

**Comment by kaygee** [comment]
Certainly. I can go back through the questions I've asked and cut/paste them on the issue you've linked.

**Comment by vikramvi** [comment]
Thanks a ton @kaygee
